homeassistant/components/binary_sensor/deconz.py

```diff
@@ -128,7 +135,7 @@ def device_info(self):
                 self._sensor.uniqueid.count(':') != 7):
             return None
         serial = self._sensor.uniqueid.split('-', 1)[0]
-        bridgeid = self.hass.data[DATA_DECONZ].api.config.bridgeid
```

Comment:
bridgeid does not make sense to me as naming, is this a copy/paste from hue-bridge?

Reply:
Deconz API is based on Hue, I want to keep local variables named consistently with how they are named in the library for easier identification. But I agree that it sounds weird.